cmd/plugin.go

```diff
@@ -82,11 +82,15 @@ Examples:
   steampipe plugin install aws
 
   # Install a specific plugin version
-  steampipe plugin install turbot/azure@0.1.0`,
+  steampipe plugin install turbot/azure@0.1.0
+
+  # Install a plugin without progress indicator
+  steampipe plugin install --progress=false aws`,
 	}
 
 	cmdconfig.
 		OnCmd(cmd).
+		AddBoolFlag(constants.ArgProgress, true, "Display control execution progress").
```

Comment:
Can you please update the description of the `--progress` flag. This seems to be the description used for `check`.

I think set it to `Display installation progress`.